examples/storybook/src/stories/ConfigureRelatedItems.stories.ts

```diff
@@ -0,0 +1,86 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('ConfigureRelatedItems', module)
+  .addDecorator(meta)
+  .add('default', () => {
+    return {
+      component: wrapWithHits({
+        template: `
+        <div>
+          <ais-index indexName="instant_search">
```

Comment:
this index does not seem to be used, and the hit itself hardcoded. If the indices are nested you should be able to reuse the hit

Reply:
If I remember correctly, the reason why the indices are not nested at the moment is you discovered some issue with nested indices and will fix it first before making them nested in stories?

Reply:
yes, however the question was for François, who made this code originally (not me)

Reply:
I don't think it's preventing this PR

Reply:
Indeed we can just replace this index by the following, it's less misleading:


```html
<div>
  <div class="ais-RelatedHits-item-image">
    <img [src]="hit.image" [alt]="hit.name" />
  </div>
  <div class="ais-RelatedHits-item-title">
    <h4>{{hit.name}}</h4>
  </div>
</div>
```